Source/SelfService/Web/documentation/container.tsx

```diff
@@ -107,6 +117,17 @@ export const DocumentationContainerScreen: React.FunctionComponent<Props> = (pro
                         <VerifyKubernetesAccess {...docProps} />
                     </Route>
 
+                    <Route exact path="/documentation/application/:applicationId/:environment/ci-cd/-azure-pipelines">
```

Comment:
```suggestion
                    <Route exact path="/documentation/application/:applicationId/:environment/ci-cd/azure-pipelines">
```

Same here with the `-` prefix.

Reply:
Fixed, good spot.